Adding links to ramp fitting docs for roman and jwst in rampviz (#3301)
spacetelescope/jdaviz: docs/rampviz/plugins.rst

Ramp Extraction
===============

Extract a ramp from a ramp cube.

Data products from infrared detectors flow through the official

> `JWST <https://jwst-pipeline.readthedocs.io/en/latest/>`_ or

Reviewer: ```suggestion
:ref:`JWST <jwst:user-docs>` or
```

> `Roman <https://roman-pipeline.readthedocs.io/en/latest/>`_ mission pipelines in levels. Infrared detectors use

Reviewer: Ideally this should also be using `:ref:` but I cannot find a good place to do it where it exist already. Might need to add one upstream or maybe use `:doc:` as you have.

Reviewer: xref https://github.com/spacetelescope/romancal/pull/1529

an "up-the-ramp" readout pattern, which is summarized in the
`JWST documentation <https://jwst-docs.stsci.edu/understanding-exposure-times>`_.

The Ramp Extraction plugin is a quick-look tool, and it does not support all of the features of the mission pipelines.
The mission pipelines produce rate images from ramp cubes by fitting the samples up the ramp while accounting for
non-linearity, jumps detected during an integration, saturation, and detector defects. These data quality checks and
corrections are not applied in the Ramp Extraction plugin. For details on how rate images are derived from ramps, see

> the JWST pipeline's :doc:`jwst:jwst/ramp_fitting/index` step or the Roman pipeline's

Reviewer: `:doc:` does not safeguard from them moving things around, so I prefer `:ref:` if possible.
```suggestion
the JWST pipeline's :ref:`jwst:ramp_fitting_step` step or the Roman pipeline's
```

> the `JWST pipeline's ramp fitting step <https://roman-pipeline.readthedocs.io/en/latest/roman/ramp_fitting/index.html>`_

Reviewer: Since you have to link to specific sections too, would intersphinx make more sense in case they move the section around in the future?

Also would `stable` be more... stable?

Author: Good ideas!

> :doc:`romancal:roman/ramp_fitting/index` step.

Reviewer: ```suggestion
:ref:`romancal:ramp_fitting_step` step.
```
